config/packages/security.yaml

```diff
@@ -18,40 +18,15 @@ security:
             security: false
 
         # Pimcore WebDAV HTTP basic // DO NOT CHANGE!
-        pimcore_admin_webdav:
-            pattern: ^/admin/asset/webdav
+        pimcore_webdav:
+            pattern: ^/asset/webdav
```

Comment:
@dvesh3 we need to change the docs accordingly as well (`pimcore/pimcore` repo).

Reply:
done https://github.com/pimcore/pimcore/pull/14971/commits/5ad1680a010b2fb3ac229fc2418a60f1262d96de#diff-82be0ccee7a311e041cdbb335fcd179e460f4cf203d7e07aeb186dce33616f2e